packages/terra-core-docs/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 
 ## Unreleased
+*Fixed
+ * Fixed Terra Arrange Reflow issue `terra-arrange`.
+ * Fixed Terra Arrange Radio button issue `terra-arrange`.
```

Comment:
change log are not clear and missing formatting

Reply:
Updated change log as per suggestion